packages/api/prisma/schema.prisma

```diff
@@ -91,6 +93,26 @@ model ProjectSummary {
   @@map("projectSummary")
 }
 
+model Vote {
+  id               Int      @id @default(autoincrement())
+  createdAt        DateTime @default(now()) @db.Timestamptz(3)
+  updatedAt        DateTime @updatedAt @db.Timestamptz(3)
+  round            Round    @relation(fields: [roundId], references: [roundId])
+  roundId          String
+  project          Project  @relation(fields: [projectId, roundId], references: [projectId, roundId])
+  projectId        String
```

Comment:
Can't projectId be derived from project ? Is this mainly for generating the unique entry ?